src/test/java/org/jenkinsci/plugins/googleplayandroidpublisher/internal/TestUtilImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.googleplayandroidpublisher.internal;
+
+import com.google.api.services.androidpublisher.AndroidPublisher;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotCredentials;
+import hudson.FilePath;
+import java.io.File;
+import java.io.IOException;
+import net.dongliu.apk.parser.bean.ApkMeta;
+import org.mockito.stubbing.Answer;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+
+public class TestUtilImpl implements JenkinsUtil, AndroidUtil {
+    public static final boolean DEBUG = true;
+
+    @Override
+    public String getPluginVersion() {
+        return "0.0.0-TEST";
+    }
+
+    @Override
+    public <R> R actOnPath(FilePath file, CheckedFunction<File, R> function) throws IOException {
+        return function.apply(new File(file.getRemote()));
+    }
```

Comment:
I'm not sure that this override is required? It seems like having the default `MasterToSlaveFileCallable` behaviour works fine?

Reply:
You're right - this does work with the existing Callable (at least on the master) 👍